### ISSUE TITLE
non-specific enzyme cleavage (Thermolysin)

### ISSUE THREAD

**Issue by fei2805** [vdemichev/DiaNN#885]
I used  non-specific enzyme cleavage (Thermolysin), and there are no options available under the protease settings. How should I set the parameters?

**Comment by vdemichev** [comment]
Non-specific searches are not really supported. What you can do is either
(i) use FragPipe (has a DIA-NN-based workflow)
(ii) prepare a non-specific digest of the sequence dabase using some other tool (e.g. R) and feed this set of peptides to DIA-NN to make a predicted spectral library and then analyse with it. Here I recommend only using DIA-NN 1.8.2 beta 27. But in any case this will result in high peptidoform false discovery rate, what would be beneficial to add some filtering of the output using MS1.Profile.Corr and Mass.Evidence.

**Comment by fei2805** [comment]
> Non-specific searches are not really supported. What you can do is either (i) use FragPipe (has a DIA-NN-based workflow) (ii) prepare a non-specific digest of the sequence dabase using some other tool (e.g. R) and feed this set of peptides to DIA-NN to make a predicted spectral library and then analyse with it. Here I recommend only using DIA-NN 1.8.2 beta 27. But in any case this will result in high peptidoform false discovery rate, what would be beneficial to add some filtering of the output using MS1.Profile.Corr and Mass.Evidence.

Thank you！
